NEWS

```diff
@@ -1,15 +1,83 @@
 #
-## Copyright (C) Mellanox Technologies Ltd. 2001-2015.  ALL RIGHTS RESERVED.
-## Copyright (C) UT-Battelle, LLC. 2014-2015. ALL RIGHTS RESERVED.
-## Copyright (C) ARM Ltd. 2017-2018.  ALL RIGHTS RESERVED.
+## Copyright (C) Mellanox Technologies Ltd. 2001-2020.  ALL RIGHTS RESERVED.
+## Copyright (C) UT-Battelle, LLC. 2014-2019. ALL RIGHTS RESERVED.
+## Copyright (C) ARM Ltd. 2017-2020.  ALL RIGHTS RESERVED.
 ##
 ## See file LICENSE for terms.
 ##
 #
 
 ## Current
+## 1.8.0-rc1 (TBD)
 Features:
-- TBD
+- Improved detection for DEVX support
+- Improved TCP scalability 
+- Added ROCM perftest
+- Added optimized memcpy for ROCM devices
+- Protection from TCP connection from non-UCX applications
+- Added hardware tag-matching for CUDA buffers
+- Added support for CUDA and ROCM managed memories
+
+Bugfixes:
+- Multiple fixes in JUCX
+- Fixes in UCP thread safety
+- Fixes for most recent versions GCC, PGI, and ICC
+- Fixes for CPU affinity on Azure instances
+- Fixes in XPMEM support on PPC64
```

Comment:
What is this?

Reply:
too minor to mention

Reply:
@yosefe People care a lot about XPMEM, worthwhile referencing

Reply:
ok